datagateway_api/src/search_api/helpers.py

```diff
@@ -74,8 +84,76 @@ def create_error_message(e):
     return wrapper_error_handling
 
 
+def get_score(entities, query, group, limit):
+    """
+    Gets the score on the given entities based in the query parameter and limit
+    """
```

Comment:
This docstring needs more details, including `:param:` and `:type:` of each parameter, following the same conventions as other docstrings in this repo